coderedcms/models/page_models.py

```diff
@@ -766,21 +766,28 @@ def upcoming_occurrences(self):
         """
         return self.query_occurrences(num_of_instances_to_return=10)
 
+
+    def _most_recent_occurrence(self):
```

Comment:
This code could probably stay where it was in the `most_recent_occurrence()` function, I don't really see the need to split it out since it should never be called on its own.